<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormToolOptionWithProgress</class>
 <widget class="QWidget" name="FormToolOptionWithProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/image/icons/view.png</normaloff>:/icons/image/icons/view.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_option_set">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/image/icons/set.png</normaloff>:/icons/image/icons/set.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_option_type">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>TYPE : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_option_id">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_option_del">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/image/icons/del.png</normaloff>:/icons/image/icons/del.png</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_run">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>â–¶</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_quality">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>311</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quality :</string>
   </property>
  </widget>
  <zorder>label_bg</zorder>
  <zorder>checkBox_view</zorder>
  <zorder>pushButton_option_set</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>pushButton_option_del</zorder>
  <zorder>checkBox_run</zorder>
  <zorder>progressBar_quality</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
